packages/web/src/components/withdraw-usdc-modal/WithdrawUSDCModal.tsx

```diff
@@ -190,7 +229,10 @@ export const WithdrawUSDCModal = () => {
           validateOnChange
           onSubmit={handleSubmit}
         >
-          {formPage}
+          <>
+            {formPage}
+            <TrackFormErrors currentBalance={balanceNumberCents} />
```

Comment:
huh i haven't seen this pattern before, mind explaining why you chose this way? Instead of say a `onChange` event handler?

Reply:
Sure, good callout as it's a bit of a weird pattern. I chose to do it this way because the only way I was aware of to get at the Formik context is to be inside a component rendered beneath that context. I didn't see any props on the top-level Formik context that let you track changes to the errors. And I wanted the sending of the event to be based on the validation error changing, not based on a separate bit of logic that watches `onChange` for a field and independently calculates its validity.